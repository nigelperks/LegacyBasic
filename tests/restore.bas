100 READ A,B
110 PRINT A,B
120 READ A,B
130 PRINT A,B
140 RESTORE 210
150 READ A,B
160 PRINT A,B
170 RESTORE
180 READ A,B
190 PRINT A,B
200 DATA 13,59
210 DATA -7,22
