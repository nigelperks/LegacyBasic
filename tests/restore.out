 13      59 
 -7      22 
 -7      22 
 13      59 
